documentation/upgrading.md

```diff
@@ -5,13 +5,34 @@ Please follow the instructions for your specific version to ensure a smooth upgr
 
 ---
 
+## Upgrading from 0.15.x to 0.16.x
+
+### 1) Deprecated `Flow\ETL\DataFrame::renameAll*` methods
+
+Methods:
+- `Flow\ETL\DataFrame::renameAll()`,
+- `Flow\ETL\DataFrame::renameAllLowerCase()`,
+- `Flow\ETL\DataFrame::renameAllUpperCase()`,
+- `Flow\ETL\DataFrame::renameAllUpperCaseFirst()`,
+- `Flow\ETL\DataFrame::renameAllUpperCaseWord()`,
+
+Were deprecated in favor of using new method: `DataFrame::renameEach()` with proper `RenameEntryStrategy` object.
+
+### 2) Deprecated `RenameAllCaseTransformer` & `RenameStrReplaceAllEntriesTransformer`
+
+Selected transformers were deprecated in favor of using `DataFrame::renameEach()` with related `RenameEntryStrategy`:
+- `RenameAllCaseTransformer` -> `RenameCaseTransformer`,
+- `RenameStrReplaceAllEntriesTransformer` -> `RenameReplaceStrategy`,
```

Comment:
I'm adding comment to create task from it to remove those deprecated classes before 1.0